docs/60-gpu.md

```diff
@@ -0,0 +1,8 @@
+# GPU
+
+Use GPUs to accelerate your Spark applications. By default, when you specify an Azure VM type that has a GPU, your Docker image will contain CUDA-8.0 and cuDNN-6.0. See [Docker Image](./12-docker-image.md) for more information about the AZTK Docker images.
```

Comment:
Update "By default, when you specify an Azure VM type that has a GPU, your Docker image will contain CUDA-8.0 and cuDNN-6.0" 

to

"When using a [GPU enabled Azure VM](https://azure.microsoft.com/en-us/pricing/details/batch/), your docker image will contain CUDA-8.0 and cuDnn-6.0 by default."